### DISCUSSION TITLE
codeblock copy

### DISCUSSION THREAD

**Discussion by lephtHanded** [enchant97/note-mark#182] [Q&A]
hey, great app!
Is there a setting or markdown format i'm missing to enable 'copy to clipboard' functionality?
```bash
the little thing to copy  way over there ------------ >
```

**Comment by lephtHanded** [comment]
nevermind, it looks like [aayushus](https://github.com/enchant97/note-mark/issues/170) already requested this back in December

**Comment by enchant97** [comment]
Closing as duplicate of #170.
